tests/test_scenarios.py

```diff
@@ -55,15 +53,16 @@ def load_scenario(path: Path) -> Tuple[Tuple[str, ...], str]:
                 assert parts[0] == "advice-animal"
                 command = tuple(parts[1:])
                 state = 1
-            elif state == 1:
+            else:
+                assert state == 1
```

Comment:
The change here is fine; originally this intended to ignore other state 0 data by falling through, and I wasn't sure if there would be a state 2 (exit code or something)

Reply:
Right, and if we get another state, we can adjust.   The old way, there was a missing branch marked.